### PR TITLE
use HEAD request to avoid rate limits

### PR DESCRIPTION
**Pull request by vito**
fixes https://github.com/concourse/concourse/issues/6074

### CODE REVIEW

File: cmd/check/main.go [hunk 1]
```diff
@@ -20,12 +19,11 @@ import (
 	ecr "github.com/awslabs/amazon-ecr-credential-helper/ecr-login"
 	ecrapi "github.com/awslabs/amazon-ecr-credential-helper/ecr-login/api"
 	"github.com/concourse/retryhttp"
-	"github.com/docker/distribution"
 	"github.com/docker/distribution/digest"
 	_ "github.com/docker/distribution/manifest/schema1"
 	_ "github.com/docker/distribution/manifest/schema2"
 	"github.com/docker/distribution/reference"
-	"github.com/docker/distribution/registry/api/v2"
+	v2 "github.com/docker/distribution/registry/api/v2"
```
Comment (xtremerui) [inline cmd/check/main.go:26]: Is this change needed?

Comment (vito) [reply cmd/check/main.go:26]: It was done automatically by `goimports` so I assume it's fine. Maybe so it doesn't get confused with go module versioning conventions which ignore the v2 suffix - in this case it's just a regular package name so it must have decided to explicitly name it that way.

**Review by xtremerui — APPROVED** [review]
